psql2mysql.py

```diff
@@ -61,6 +61,37 @@ def _exclude_deleted(self, table, query):
             return query.where(table.c.deleted == 'False')
         return query.where(table.c.deleted == False)
 
+    def getTextColumns(self, table):
+        columns = table.columns
+        return [ c.name for c in columns if str(c.type) == 'TEXT' ]
+
+    def scanTableForLongTexts(self, table):
+        textColumns = self.getTextColumns(table)
+        if not textColumns:
+          return []
+        LOG.debug("Scanning Table %s (columns: %s) for too long TEXT values ",
+                 table.name, textColumns)
+        filters = [
+            text("length(\"%s\") > %i" % (x, MAX_TEXT_LEN)) for x in textColumns
+        ]
+        q = table.select().where(or_(f for f in filters))
+        rows = q.execute()
+
+        long_values = []
+        primary_keys = []
+        if not table.primary_key == None:
+          primary_keys = list(table.primary_key)
```

Comment:
I guess that `table.primary_key` is a single element? if so is more clear this: `[table.primary_key]`

If can be a single element, or an iterator, I think that our code is OK as in any case will return a list.

Reply:
No, there could be more